src/libraries/Microsoft.PowerFx.Core/Entities/External/IExternalCdsDataSource.cs

```diff
@@ -15,5 +17,7 @@ internal interface IExternalCdsDataSource : IExternalTabularDataSource
         IExternalTableDefinition TableDefinition { get; }
 
         bool TryGetRelatedColumn(string selectColumnName, out string additionalColumnName, IExternalTableDefinition expandsTableDefinition = null);
+
+        bool IsArgTypeValidForMutation(DType type, out IEnumerable<string> invalidFieldName);
```

Comment:
>IsArgTypeValidForMutation [](http://example.com/codeflow?start=13&length=25)

Curious what's implementing this? Parser code?

Reply:
DataverseDataSourceInfo